src/signals/settings/users/containers/Overview/hooks/useFetchUsers.js

```diff
@@ -25,16 +25,33 @@ const useFetchUsers = ({ page, pageSize } = {}) => {
     (async function fetchData() {
       setLoading(true);
 
+      const pageParams = [
+        page && `page=${page}`,
+        pageSize && `page_size=${pageSize}`,
+      ];
+
+      const filterParams = Object.keys(filters || {}).reduce((acc, filter) => {
```

Comment:
I think this can be simplified a bit:
```jsx
const filterParams = Object.entries(filters || {}).reduce(
  (acc, [filter, value]) => [...acc, `${filter}=${value}`],
  []
);
```

and maybe even

```jsx
const params = [
  page && `page=${page}`,
  pageSize && `page_size=${pageSize}`,
  Object.entries(filters || {}).reduce(
    (acc, [filter, value]) => [...acc, `${filter}=${value}`],
    []
  ),
]
  .filter(Boolean)
  .join('&');
```